packages/fx-core/src/plugins/resource/sql/utils/dialogUtils.ts

```diff
@@ -16,12 +16,13 @@ export class DialogUtils {
 
 export class ProgressTitle {
   static readonly Provision = "Provisioning SQL";
-  static readonly ProvisionSteps = 2;
+  static readonly ProvisionSteps = 3;
```

Comment:
Better to get the number of steps according to the messages so that we don't need to update the number every time we add/remove a step.

Reply:
It is fixed. Thanks.